lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +232,24 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        container_div = article_bs.find('div', class_='col-lg-9 col-md-12')
+        for link in container_div.find_all('a'):
+            if link.get('href').startswith('/news/202'):
+                href = 'https://www.gazetavechorka.ru' + link.get('href')
+                if href and href not in self.urls:
+                    self.urls.append(href)
+                    return href
 
     def find_articles(self) -> None:
         """
         Find articles.
         """
+        seed_urls = self.config.get_seed_urls()
+        for url in seed_urls:
+            response = make_request(url, self.config)
+            if response.status_code < 400:
+                soup = BeautifulSoup(response.text, 'html.parser')
```

Comment:
how do you control number of articles that you collect and stop if they are collected

Reply:
fixed